vcpkg/commands/hash.md

```diff
@@ -0,0 +1,41 @@
+---
+title: vcpkg hash
+description: Reference for the vcpkg hash command. Computes the hash of a specified file.
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 08/17/2023
+ms.prod: vcpkg
+---
+
+
+# vcpkg hash
+
+## Synopsis
+
+```console
+vcpkg hash <file-to-hash> [algorithm]
+```
+## Description
+
+Computes the hash of a specified file. By default, it computes the SHA-512 hash, but another algorithm can be optionally specified. The output will be the computed hash for the given file displayed directly on the console.
+
+>[!NOTE]
+>The command requires two arguments:
+>1. A mandatory file path.
+>2. An optional hash algorithm, either `SHA256` or `SHA512`. If omitted, it will default to using SHA512 to compute the file's hash.
```

Comment:
1. This is not what requires means.
2. This is not noteworthy above and beyond the synopsis.